prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/aggregations/maia.yaml

```diff
@@ -1,33 +1,83 @@
+    - record: netapp_volume_snapshot_available_bytes:pre
+      expr: label_replace(netapp_volume_snapshot_reserve_available{app="netapp-harvest-exporter-manila"}, "vserver", "$1", "svm", "(.*)")
+
+    - record: netapp_volume_snapshot_used_bytes:pre
+      expr: label_replace(netapp_volume_snapshots_size_used{app="netapp-harvest-exporter-manila"}, "vserver", "$1", "svm", "(.*)")
+
+    - record: netapp_volume_inode_files_total:pre
+      expr: label_replace(netapp_volume_inode_files_total{app="netapp-harvest-exporter-manila"}, "vserver", "$1", "svm", "(.*)")
+
+    - record: netapp_volume_inode_files_used:pre
+      expr: label_replace(netapp_volume_inode_files_used{app="netapp-harvest-exporter-manila"}, "vserver", "$1", "svm", "(.*)")
+
+    #
+    # performance metrics
+    #
+
+    - record: netapp_volume_total_ops:pre
+      expr: netapp_volume_total_ops{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    - record: netapp_volume_read_ops:pre
+      expr: netapp_volume_read_ops{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    - record: netapp_volume_write_ops:pre
+      expr: netapp_volume_write_ops{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    # volume read throughput in bytes per second
+    - record: netapp_volume_read_throughput:pre
+      expr: netapp_volume_read_data{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    # volume write throughput in bytes per second
+    - record: netapp_volume_write_throughput:pre
+      expr: netapp_volume_write_data{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    - record: netapp_volume_read_latency_milliseconds:pre
+      expr: netapp_volume_read_latency{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+    - record: netapp_volume_write_latency_milliseconds:pre
+      expr: netapp_volume_write_latency{app="netapp-harvest-exporter-manila"} * on (filer, svm, volume) group_left(project_id, share_id, share_name, share_type) netapp_volume_labels{app="netapp-harvest-exporter-manila"}
+
+
+    #
+    # only keep necessary labels visible to users
+    #
+
+    # We need to take average of the metrics scraped from different exporters, to avodid exporting multiplied values
```

Comment:
'different exporters' here means having multiple pods of harvest-exporter that would export from the same filer, right?

Reply:
maybe `multiple harvest worker` is easier to read. I meant that it is possible same filer is scraped by multiple harvest workers due to some errors in netappsd.

Reply:
Yes, that would make it clearer 👍